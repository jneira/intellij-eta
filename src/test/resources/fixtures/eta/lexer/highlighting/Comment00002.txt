ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Comment00002')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITblockComment ('{-')
ITblockComment (' NOTES:\n\nThe trick of doing a copy at the end, i.e. `tellProg [copyProg loc\ne]`, when compiling WithArray is important.\n')
ITblockComment ('-}')
