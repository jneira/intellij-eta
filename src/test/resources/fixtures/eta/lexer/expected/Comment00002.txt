ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Comment0000')
WHITE_SPACE ('2')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITblockComment ('{- NOTES:\n\nThe trick of doing a copy at the end, i.e. `tellProg [copyProg loc\ne]`, when compiling WithArray is important.\n-}')
WHITE_SPACE ('')
ITvocurly ('')